application/src/main/java/net/corda/v5/application/crypto/SigningService.java

```diff
@@ -57,4 +57,44 @@ public interface SigningService {
     @Suspendable
     @NotNull
     Map<PublicKey, PublicKey> findMySigningKeys(@NotNull Set<PublicKey> keys);
+    
+    /**
+     * Deserializes a {@link PublicKey} from a {@code byte} array.
+     *
+     * @param encodedKey The public key represented asa a {@code byte} array.
+     * @return An instance of {@link PublicKey} constructed from the encoded key.
+     */
+    @Suspendable
```

Comment:
In two minds whether these should be Suspendable, they will never actually suspend, but marking them as such opens up to future changes - however I see other functions in this module are not all marked as suspendable so maybe we should remove that?

Reply:
Removed suspendable annotation.